tests_e2e/playwright/.env.tpl

```
@@ -0,0 +1,3 @@
+USERNAME="USERNAME_OR_GOOGLE_EMAIL"
+PASSWORD="PASSWORD"
```

These should be 

```suggestion
KEYCLOAK_USERNAME="USERNAME_OR_GOOGLE_EMAIL"
KEYCLOAK_PASSWORD="PASSWORD"
```

to avoid name clashes.